dist/bin/scala

```diff
@@ -82,9 +92,12 @@ while [[ $# -gt 0 ]]; do
       shift ;;
     *)
       if [ $execute_script == false ]; then
-        if [[ "$1" == *.scala ]]; then
+        # is a script if extension .scala or .sc or if has scala hash bang
```

Comment:
This conflicts with the definition of `isScript` in `SourceFile` which says that a script must not have an extension `*.scala`.

Reply:
good catch, that was intended for a possibly slight performance gain, but might not have actually provided one anyway.